setup.py

```diff
@@ -4,18 +4,21 @@
 from setuptools import setup, find_packages
 
 setup(
-    name='canonicalwebteam.http',
-    version='0.1.6',
-    author='Canonical webteam',
-    author_email='webteam@canonical.com',
-    url='https://github.com/canonicalwebteam/http',
+    name="canonicalwebteam.http",
+    version="0.1.7",
```

Comment:
Didn't we talk about making this `1.0.0`? I think this is a fairly fundamental change to how the module works, and probably warrants a new major version. Or if you don't think we're ready for v1 yet, we could make it `0.2.0`, but it certainly feels like more than a point release.

Reply:
Youre right. This slipped my mind. Updated